tensorflow/lite/micro/kernels/softmax.cc

```diff
@@ -30,23 +30,38 @@ namespace micro {
 namespace activations {
 namespace {
 
+// Softmax parameter data that persists in user_data
+static constexpr int size_of_lut = 513;
+struct SoftmaxOpData {
```

Comment:
Can you use exp_lut and one_over_one_plus_x_lut from SoftmaxParams instead of creating this new wrapper struct? See the implementation in micro/kernels/xtensa_hifimini/softmax.cc for reference.

Reply:
Done.
Yeah this wrapper struct is quite redundant. :)